nft-simple/src/internal.rs

```diff
@@ -1,5 +1,7 @@
 use crate::*;
 
+const YOCTO_MULTIPLIER: f32 = 0.000000000000000000000001;
```

Comment:
why do we use floats?

Reply:
I wanted a clear error that's helpful to folks, is there a better way I should do this? Pitfalls or advice welcome.

<img width="1348" alt="Screen Shot 2021-03-26 at 9 16 04 AM" src="https://user-images.githubusercontent.com/1042667/112661723-0c319c00-8e14-11eb-97cc-135fe3fb6508.png">

Reply:
removed